src/core/net/ip6.cpp

```diff
@@ -184,6 +187,8 @@ ThreadError Ip6::InsertMplOption(Message &aMessage, Header &aIp6Header, MessageI
 {
     ThreadError error = kThreadError_None;
 
+    otLogFuncEntry();
```

Comment:
Are these logs really necessary? For debugging, it does not look like a useful message at all, since every dbg tool can provide that.

From high-level thread standpoint, when running in debug mode we are more interested in thread-specific actions, not if code enters a function or not.